lib/idx/factory/api.ts

```diff
@@ -39,9 +39,16 @@ import {
 } from '../transactionMeta';
 import { FlowIdentifier, IdxAPI, OktaAuthIdxInterface } from '../types';
 import { unlockAccount } from '../unlockAccount';
+import * as remediators from '../remediators';
+import { getFlowSpecification } from '../flow/FlowSpecification';
+import { setRemediatorsCtx } from '../util';
 
 // Factory
 export function createIdxAPI(sdk: OktaAuthIdxInterface): IdxAPI {
+  setRemediatorsCtx({
```

Comment:
See `setRemediatorsCtx ` in utils - https://github.com/okta/okta-auth-js/pull/1476/files#diff-71b56cc120e09a7c5f1d47c7637461aece91f39807b35359697038d48b38ddd3
This is needed for minimal IDX API to not import all remediators and flow specifications (except GenericRemediator).
By default `remediatorsCtx` has empty list of remediators